ui/apps/platform/src/Containers/Policies/Wizard/Step4/PolicyScopeForm.tsx

```diff
@@ -178,7 +196,7 @@ function PolicyScopeForm(): ReactElement {
                     <FlexItem className="pf-v5-u-pr-md" alignSelf={{ default: 'alignSelfCenter' }}>
                         <Button
                             variant="secondary"
-                            isDisabled={!hasDeployOrRuntimeLifecycle}
+                            isDisabled={!hasDeployOrRuntimeLifecycle || isAllScopingDisabled}
```

Comment:
Redundant, but reduces the possibility of a future change breaking this unintentionally.